cypress/e2e/seededFlows/adminFlows/articles/privilegedReactions.spec.js

```diff
@@ -0,0 +1,181 @@
+  });
+
+  it('should display the correct vomit count', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '0');
+  });
+
+  it('should display the correct score', () => {
+    cy.get('.flex .crayons-card:nth-child(4) .fs-s').should('contain', '0');
+  });
+});
+
+describe('Article flagged by a user', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.intercept('POST', '/admin/reactions/*', (req) => {
+      req.reply((res) => {
+        const { id } = req.body;
+        const response = {
+          statusCode: 200,
+          body: {
+            outcome: 'Success',
+          },
+        };
+
+        if (req.body.removeElement === true) {
+          cy.get(`#${id}`).then(($element) => {
+            $element.remove();
+          });
+          cy.get(`#js__reaction__div__hr__${id}`).then(($element) => {
+            $element.remove();
+          });
+        } else {
+          cy.reload();
+        }
+
+        res.send(response);
+      });
+    });
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/admin/content_manager/articles/1'); // This article contains one flagged reaction.
+      });
+    });
+  });
+
+  it('should display the correct default data', () => {
+    cy.get('.flex .crayons-card:nth-child(3) .fs-s').should('contain', '1');
+  });
+
+  it('should display the correct user name', () => {
+    cy.get('.flex .crayons-subtitle-3').should(
+      'contain',
+      'Trusted User 1 \\:/',
+    );
+  });
+
+  it('should display the open flag status', () => {
+    cy.get('.flex .c-indicator').should('be.visible').as('flagStatus');
+    cy.get('@flagStatus')
+      .should('have.class', 'c-indicator--warning')
+      .and('contain', 'Open');
+  });
+
+  it('should display the correct score for open flag', () => {
+    cy.get('.flex .crayons-card:nth-child(2) .fs-s').should('contain', '-50');
+  });
+
+  it('should open the dropdown on button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+  });
+
+  it('should open the dropdown on button click and close the dropdown on same button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('not.visible');
+  });
+
+  it('should open the dropdown on button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+  });
+
+  it('should open the dropdown on button click and close the dropdown on same button click', () => {
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('be.visible');
+
+    cy.get('.c-btn--icon-alone').click();
+    cy.get('.crayons-dropdown').should('not.visible');
+  });
+
```

Comment:
Are these test duplicate of the above?

Reply:
They were partially duplicate which I have update now. But the main point is that in first test we are performing single click on button to open the dropdown and in second we are clicking it twice to make sure that the dropdown closes too.